### ISSUE TITLE
Markdown editor sometimes not save the data

### ISSUE THREAD

**Issue by rubenrcdev** [laravel-filament/filament#13385]
### Package

filament/forms

### Package Version

v3.2.92

### Laravel Version

v11.11.1

### Livewire Version

v3.5.1

### PHP Version

PHP 8.3.8

### Problem description

Hi,

https://github.com/filamentphp/filament/assets/120738508/47c6f253-8648-4f35-8360-124240126748

When you update a resource with the Markdown Editor, sometimes the new data is not saved.
I tried to find the problem before send the bug report, for trying to elucidate the reason why it occurs. But not was possible.
I only detected, that when the resource is not saved, the ajax requests, not have the "updated" field filled about the current form state. Is like, it dont detect the changes in the form state.

I let you a video about the bug, and i let you a demo repo, is a very basic filament panel, so, is not a bug in a complex system, it occurs even in an panel with a one resource, one model and two fields in a model.

Thank you for all.

### Expected behavior

Save the data all the times.

### Steps to reproduce

Enter o reload resource page, and try to save multiple times the form, sometimes is saved, other times not.

### Reproduction repository (issue will be closed if this is not valid)

https://github.com/rubenrcdev/filament-editor-bug

**Comment by rubenrcdev** [comment]
I think is related with the issue #12858

**Comment by rubenrcdev** [comment]
I did more testing to find what happens in the markdown editor.
 
I modified `public/js/filament/forms/components/markdown-editor.js` file, to include a console.log as can see in the next screenshoot:
<img width="462" alt="image" src="https://github.com/filamentphp/filament/assets/120738508/1e5c87d6-0d8f-4baf-8703-300a2cc6b205">

When i write something, the watch state takes some time to refresh data, it takes about 1 second to refresh, if i save before that happens, the data is not updated.

The bug is not from filament, is about how works livewire.
So, i close the issue.

Thanks, and regards.
